### PR TITLE
Api fetch and send data to store.  Only once.

### PR DESCRIPTION
**Pull request by Meltrust**
# In this milestone I:

- Hit the API for rockets data.
- Updated the redux store with only the data that was requested.
- Made sure the store updated only once.
- Used useSelector and useDispatch.

### CODE REVIEW

**Comment by AlexHamn** [comment]
Hi @Meltrust, 

Good job so far!
I can see you're really putting effort to become a world-class software developer :+1:
There are some issues that you still need to address in order to move forward to the next project.
Trust me you got this! :muscle:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Consider the optional changes nice to have features and only work on them when you have satisfied the required changes_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable

File: src/redux/configureStore.js [hunk 1]
```diff
@@ -2,8 +2,8 @@ import { createStore, combineReducers, applyMiddleware } from 'redux';
 import { composeWithDevTools } from 'redux-devtools-extension';
 import logger from 'redux-logger';
 import thunk from 'redux-thunk';
-import missionsReducer from './rockets/rockets';
-import rocketsReducer from './missions/missions';
+import rocketsReducer from './rockets/rockets';
+import missionsReducer from './missions/missions';
```
Comment (AlexHamn) [inline src/redux/configureStore.js]: * [x]  [REQUIRED] Kindly import the `missionsReducer` first since it is the superior reducer.

**Review by AlexHamn — CHANGES_REQUESTED** [review]
Hi @Meltrust, 

Good job so far!
I can see you're really putting effort to become a world-class software developer :+1:
There are some issues that you still need to address in order to move forward to the next project.
Trust me you got this! :muscle:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Consider the optional changes nice to have features and only work on them when you have satisfied the required changes_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable

File: src/redux/configureStore.js (hunk 1 above)
Comment (Meltrust) [reply src/redux/configureStore.js]: Done!

**Review by AlexHamn — APPROVED** [review]

**Pull request merged**